src/documents/activities/StatesandContextsMapping.html.md.handlebars

```diff
@@ -0,0 +1,42 @@
+---
+title: States and Contexts Mapping
+layout: content
+category: Activities
+---
+
+This activity breaks down the User States and Contexts tool into modular pieces and actionable steps, and it is also a re-thinking of the original User States and Contexts tool.
+
+Here, states are reframed as functional requirements. Deficit language or disability classifications that are functionally ambiguous are avoided.
+
+Placing needs that are commonly met in the middle and needs that are not well met further out, demonstrates that edge needs are greater in number, and communicates the phenomena that by meeting edge needs, you also meet the needs in the middle.
+
+In this activity, the states and contexts facets can be swapped in and out depending on the context and the design challenge. Facets can also be broken down into more specific considerations. For example, cognitive access might include memory, focus and other considerations. 
+
+This activity is intended to be used in the context of co-design where a diverse set of co-designers, including people who presently have difficulty using the design, are active participants in the design process. Co-designers are able to create their own facets if they can't find a desired state or context within the provided set.
+
+<ul>
+<li>The exercise is about <strong>stretching and reaching</strong>: to include people who are currently at the edges and who are therefore usually excluded from the design process.</li>
```

Comment:
"Edges" may be ambiguous? I was thinking of the edge of the petal, but could also mean an "edge case". Maybe they're both the same, but I'm not 100% sure.